doc/manual/src/contributing/hacking.md

```diff
@@ -108,3 +108,72 @@ These tests include everything that needs to interact with external services or
 Because these tests are expensive and require more than what the standard github-actions setup provides, they only run on the master branch (on <https://hydra.nixos.org/jobset/nix/master>).
 
 You can run them manually with `nix build .#hydraJobs.tests.{testName}` or `nix-build -A hydraJobs.tests.{testName}`
+
+## Testing the install scripts
+
+Testing the install scripts has traditionally been tedious, but you can now do this much more easily via the GitHub Actions CI runs (at least for platforms that Github Actions supports).
+
+If you've already pushed to a fork of Nix on GitHub before, you may have noticed that the CI workflows in your fork list skipped `installer` and `installer_test` jobs. Once your Nix fork is set up correctly, pushing to it will also run these jobs.
```

Comment:
```suggestion
Testing the install scripts is currently most convenient with GitHub Actions.

The Nix repository has [two continuous integration (CI) workflows](https://github.com/NixOS/nix/blob/88a45d6149c0e304f6eb2efcc2d7a4d0d569f8af/.github/workflows/ci.yml#L50-L91), which are run every time changes are pushed to a branch:

```

Reply:
I realize this will grow crusty over time and will be moot if the ~binary installer project reaches fruition, but I'm trying to drop breadcrumbs that encourage people who attempted some small installer fix/improvement in the past (only to be scarred by the testing/review ~process) to take a second look.

Reply:
I have a reworked draft of the section covered by this edit and the one below, but I'll hold off on committing and pushing until we hash out the title question.